incubator/haproxy-ingress/templates/controller-daemonset.yaml

```diff
@@ -182,6 +190,10 @@ spec:
 {{ toYaml .Values.controller.affinity | indent 8 }}
     {{- end }}
     {{- if .Values.controller.priorityClassName }}
-      priorityClassName: {{ .Values.controller.priorityClassName }}
+      priorityClassName: {{ .Values.controller.priorityClassName | quote }}
+    {{- end }}
+    {{- if .Values.controller.securityContext }}
+      securityContext:
+{{ toYaml .Values.controller.securityContext | indent 8 }}
```

Comment:
Cloud you provide some document

Reply:
`securityContext` was being set in controller-deployment.yaml, but was missing in controller-daemonset.yaml.